cargo-dist/templates/installer/installer.ps1.j2

```diff
@@ -55,6 +55,18 @@ $receipt = @"
 "@
 $receipt_home = "${env:LOCALAPPDATA}\{{ app_name }}"
 
+if ($env:{{ disable_update_env_var }}) {
```

Comment:
We need to be checking INSTALLER_INSTALL_UPDATE(R) as well here.

Reply:
disregard this, we are explicitly *not* adding generic INSTALLER_ vars in this PR